docs/first_objective.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>This should also have a legend explaining which color represents which element</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I've added the legend now</p>

Reply:
<p>Also, consider a different y-axis scale since the less abundant elements are all squashed into the bottom of the figure and it is not informative</p>

Reply:
<p>Done, changes the y-scale to log</p>